iceoryx_hoofs/test/moduletests/test_cxx_string.cpp

```diff
@@ -1748,8 +1796,9 @@ TYPED_TEST(stringTyped_test, CompareWithStdStringResultPositiveWithDifferentSize
     ::testing::Test::RecordProperty("TEST_ID", "08891d54-3db7-47cd-8e42-9beb7035c044");
     using MyString = typename TestFixture::stringType;
     constexpr auto STRINGCAP = MyString::capacity();
-    std::string testString(STRINGCAP + 6U, 'M');
-    string<STRINGCAP + 6U> sut;
+    constexpr auto STRINGCAP_PLUS6 = STRINGCAP + 6U;
+    std::string testString(STRINGCAP_PLUS6, 'M');
+    string<STRINGCAP_PLUS6> sut;
```

Comment:
This is also just to satisfy a tool. We used to complain about this. What happened to us :smile:

Reply:
You're right, shame on me. I removed most of the unnecessary constexpr.